### ISSUE TITLE
Green Skin in MainTex in ver. 6.4.1

### ISSUE THREAD

**Issue by VP1989** [FlailingFog/KK-Blender-Porter-Pack#383]
1st card - ![Koikatu_F_20201107214831593_Yuzuriha (Big Areolas)](https://user-images.githubusercontent.com/8047059/220713855-8cc66e74-d986-4656-af2e-a80717d9cd36.png)

The other card - https://files.catbox.moe/mhx4mb.png

I have been noticing that when I import a Koikatsu model to Blender, they  end up having green skin, which is not in the real maintex.

![image](https://user-images.githubusercontent.com/8047059/220714396-10ed9447-eb6b-4851-9114-caf372ecda83.png)

![image](https://user-images.githubusercontent.com/8047059/220714524-19b2ba84-2c98-4536-9386-a6e7e4bbf9b6.png)

**Comment by FlailingFog** [comment]
Sorry, not sure what could cause that. This is what I'm getting:
![image](https://user-images.githubusercontent.com/65811931/220793998-a1087927-7c87-4363-a59c-c8ccb9bf811f.png)

**Comment by VP1989** [comment]
I thought the issue was with Blender. I reinstalled it. Still green skin.

Then I realised I was using an old version of the KKBP Exporter plugin. It was 640 instead of 642. Installed 642, but the issue was not fixed.

Then I tried redownloading the KKBP Exporter DLL and tried a different card this time.

The new card - https://www.pixiv.net/en/artworks/100373659

The issue still persists and somehow a different maintex is still created each time.

![image](https://user-images.githubusercontent.com/8047059/220985371-b4e9df19-1977-4fea-b5e4-5449529f8621.png)

I am using Blender 3.4.1. This issue has not happened before, but started happening only recently. I even tried reinstalling Blender 3.4.0, but it did not make a difference.

What other details should I provide?
